multiplatform-markdown-renderer-m2/src/commonMain/kotlin/com/mikepenz/markdown/m2/MarkdownTypography.kt

```diff
@@ -23,9 +26,14 @@ fun markdownTypography(
     paragraph: TextStyle = MaterialTheme.typography.body1,
     ordered: TextStyle = MaterialTheme.typography.body1,
     bullet: TextStyle = MaterialTheme.typography.body1,
-    list: TextStyle = MaterialTheme.typography.body1
+    list: TextStyle = MaterialTheme.typography.body1,
+    link: TextStyle = MaterialTheme.typography.body1.copy(
+        fontWeight = FontWeight.Bold,
+        color = markdownColor().linkText,
```

Comment:
This is potentially a different color from what `LocalMarkdownColors.current.linkText` ultimately holds.

Reply:
Please remove the color here, and apply it in place where the link is used, as it was defined before.

Reply:
removed the color in m2 and m3 typography creation and moved it the `AnnotatedString` extensions